recipes/gdk-pixbuf/all/conanfile.py

```diff
@@ -130,12 +157,20 @@ def _configure_meson(self):
         defs["builtin_loaders"] = "all"
         defs["gio_sniffing"] = "false"
         defs["introspection"] = "enabled" if self.options.with_introspection else "disabled"
-        args=[]
+        args = []
+        # Workaround for https://bugs.llvm.org/show_bug.cgi?id=16404
+        # Ony really for the purporses of building on CCI - end users can
+        # workaround this by appropriately setting global linker flags in their profile
+        if self._requires_compiler_rt:
+            args.append('-Dc_link_args="-rtlib=compiler-rt"')
         args.append("--wrap-mode=nofallback")
         meson.configure(defs=defs, build_folder=self._build_subfolder, source_folder=self._source_subfolder, pkg_config_paths=".", args=args)
         return meson
 
     def build(self):
+        if self._requires_compiler_rt:
+            self._test_for_compiler_rt()
+
```

Comment:
maybe add this in validate function

Reply:
I did try that but unfortunately compiler information is not available to conan at that point.

I'm not sure what the best fix for this is - it has come up before. I might propose an upstream fix to glib to define `__muloti4` itself if it isn't available but I'm not that familiar with the meson build system.